crates/bevy_pbr/src/lib.rs

```diff
@@ -2,8 +2,18 @@
 #![allow(missing_docs)]
 #![cfg_attr(docsrs, feature(doc_auto_cfg))]
 
+mod meshlet;
 pub mod wireframe;
 
+/// Experimental features that are not yet finished. Please report any issues you encounter!
+///
+/// Expect bugs, missing features, compatibility issues, low performance, and/or future breaking changes.
+pub mod experimental {
+    pub mod meshlet {
```

Comment:
I think it _probably_ makes sense to put this behind a cargo feature. This is thousands of lines of code that most people won't be using yet, and that a good chunk of 3D devs will never use even once it leaves experimental (for platform support / runtime constraint reasons). The code already seems separate enough that I think we can swing this. And putting this behind a flag also draws hard lines that ensures the experimental code doesn't become "entangled" with non-experimental code.

Reply:
Which parts specifically do you want to be behind a feature flag? Just the meshlet module? What about the material API parts, like the meshlet material functions?

It'll also get a bit tricky due to the meshlet example then needing a non-default feature.

Reply:
I think it would be fine to only hide the module behind a flag, but I guess if we do that might as well add the flag to the material extensions. I don't think it's a big issue for the example, we already have examples that need specific features.

Reply:
> Which parts specifically do you want to be behind a feature flag? Just the meshlet module? What about the material API parts, like the meshlet material functions?

Yeah I think both the module and the material API parts are the move. We could get away with keeping the material API stuff, but I think its cleaner to also block that out.

And yeah we can just add the `meshlet` feature to the required features list for the example.